lib/types/user-types.ts

```diff
@@ -1,5 +1,16 @@
+import { Privilege } from "@prisma/client";
+
 export interface FormOwner {
   id: string;
   email: string;
+  name?: string;
+  active: boolean;
+}
+
+export interface DBUser {
```

Comment:
FYI it is also possible to import types from Prisma for the defined models.
Would the Prisma User type meet your needs?
`import { User } from "@prisma/client";`

Reply:
Almost I updated the code to extend the prisma type

```
export interface DBUser extends User {
  privileges: Privilege[];
}
```